gems/aws-sdk-core/spec/aws-sdk-core/middleware/checksums_spec.rb

```diff
@@ -0,0 +1,234 @@
+# frozen_string_literal: true
+
+require_relative '../../spec_helper'
+require 'hearth'
```

Comment:
I would prefer if we can stub instead of requiring hearth. It also would not be a dependency of the library and these tests would only succeed when installed locally or with -I

Reply:
NVM